cpp/custrings/text/edit_distance.cu

```diff
@@ -227,3 +257,61 @@ unsigned int NVText::edit_distance( distance_type algo, NVStrings& strs1, NVStri
     return 0;
 }
 
+unsigned int NVText::edit_distance_matrix( distance_type algo, NVStrings& strs, unsigned int* results, bool bdevmem )
+{
+    if( algo != levenshtein || results==0 )
```

Comment:
- Use `CUDF_EXPECTS` for checking expectations (will throw an exception)
 - Use `nullptr`, not `0` for pointer comparisons.
 - It seems like an unsupported algorithm and a nullptr should be two separate failed expectations.

  ```suggestion
    CUDF_EXPECTS(algorithm == levenshtein, "Unsupported algorithm");
    CUDF_EXPECTS(results != nullptr, "Null results pointer");
```